tests/serenity/src/test/java/bla/tm/definitions/site/pantheon/UserData.java

```diff
@@ -0,0 +1,16 @@
+package bla.tm.definitions.site.pantheon;
+
+/**
+ * Created by Maksym_Mazurkevych on 3/16/2017.
+ */
+public class UserData {
+
+    public UserData(String username, String password) {
+        this.username = username;
+        this.password = password;
+    }
+
+    public String username;
+    public String password;
+
+}
```

Comment:
I suggest create some file with all passwords and then expand this class with logic when user send name of object and then can get appropriate logic and password from this object
because now this class looks redundant